src/tests/ftest/pool/evict.py

```diff
@@ -56,204 +53,160 @@ def connected_pool(self, hostlist, targets=None):
-            self.log.info("Expected exception - invalid param %s\n %s\n",
-                          test_param, str(result))
-
-            # verify that pool still exists and the handle is still valid.
-            self.log.info("Check if pool handle still exist")
-            return self.pool_handle_exist(self.pool, test_param)
-
-        # if here then pool-evict did not raise an exception as expected
-        # restore the valid server group name and check if valid pool
-        # still exists
-        self.log.info(
-            "Command exception did not occur"
-            " - evict from pool with %s", test_param)
-
-        # check if pool handle still exists
-        self.pool_handle_exist(self.pool, test_param)
-
-        # Commented out due to DAOS-3836.
-        # if self.pool.check_files(self.hostlist_servers):
-        #    self.log.error("Valid pool files were not detected on server after"
-        #                   " a pool evict with %s failed to raise an "
-        #                   "exception", test_param)
-        self.log.error("Test did not raise an exception with when "
-                       "evicting a pool with bad param: %s", test_param)
-        return False
+        if pool.dmg.result.exit_status != 0:
+            self.fail("Pool evict failed!")
 
     def test_evict(self):
         """
-        Test evicting a client from a pool.
-
-        Test creates 2 pools on 4 target (pool_tgt [0,1,2,3])
-        and 1 pool on 2 targets (pool_tgt_ut [0,1]).  The pools are connected
-        to and a container with data is created on all 3.
-        The evict is done on connection to the pool with 2 targets.
-        The handle is removed.
-        The test verifies that the other two pools were not affected
-        by the evict
+        Test Steps:
+        1. Create 2 pools on 4 server ranks; rank 0 to 3.
+        2. Create another pool on 2 ranks; rank 0 and 1.
+        3. After creating each pool, create a container and write objects.
+        4. Verify that the pool file exists in /mnt/daos.
+        5. Verify that the third pool's file doesn't exist on the two hosts that this
+        pool wasn't created on.
+        6. Evict the third pool on rank 0 and 1.
+        7. Verify that the pool file still exists in /mnt/daos for all three pools. i.e.,
+        verify that the evict didn't cause the ill effect to the pool file.
+        8. For all pools, call pool_query() over API. The first two pools should
+        succeed. The last pool should fail because it was evicted.
+        9. For the first two pools, check the UUID obtained from the pool_query() in
+        the previous step and verify that the evict on the third pool didn't have ill
+        effect.
 
         :avocado: tags=all,pr,daily_regression
-        :avocado: tags=small
-        :avocado: tags=pool,pool_evict,pool_evict_basic
+        :avocado: tags=vm
+        :avocado: tags=pool
+        :avocado: tags=pool_evict,pool_evict_basic
```

Comment:
FYI, these tags are more inline with the comment I made on the pool destroy PR. I.e. if we use `pool_destroy`, it's more inline with `pool_evict` here.

Reply:
Done. Thanks